README.md

```diff
@@ -15,24 +15,24 @@ This list contains only stage 1 proposals and higher that have not yet been with
-| :rocket: | [`s` (`dotAll`) flag for regular expressions][dot-all]                           | Mathias Bynens                              | Brian Terlson, Mathias Bynens                   | [:white_check_mark:](https://github.com/tc39/test262/issues/997)  |
-|          | [Legacy RegExp features in JavaScript][regexp-legacy]                            | Claude Pache                                | Mark Miller, Claude Pache                       | [:white_check_mark:](https://github.com/tc39/test262/issues/1165) |
-|          | [`Promise.prototype.finally`][finally]                                           | Jordan Harband                              | Jordan Harband                                  | [:construction:](https://github.com/tc39/test262/issues/866)      |
-|          | [`BigInt`][bigint]                                                               | Daniel Ehrenberg                            | Daniel Ehrenberg                                | [:construction:](https://github.com/tc39/test262/issues/1056)     |
-|          | [Optional `catch` binding][optional-catch]                                       | Michael Ficarra                             | Michael Ficarra                                 | [:white_check_mark:](https://github.com/tc39/test262/issues/1166) |
-|          | [`import.meta`][import-meta]                                                     | Domenic Denicola                            | Domenic Denicola                                | [:question:](https://github.com/tc39/test262/issues/1342)         |
-|          | [Private methods and accessors][private-methods]                                 | Daniel Ehrenberg                            | Daniel Ehrenberg, Kevin Gibbons                 | [:question:](https://github.com/tc39/test262/issues/1343)         |
-|          | [`Array.prototype.{flatMap,flatten}`][flat]                                      | Brian Terlson, Michael Ficarra              | Brian Terlson, Michael Ficarra                  | [:question:](https://github.com/tc39/test262/issues/new)          |
-|          | [Numeric separators][numeric_separators]                                         | Sam Goto, Rick Waldron                      | Sam Goto, Rick Waldron                          | [:construction:](https://github.com/tc39/test262/issues/1051)     |
-|          | [Class Public Instance Fields & Private Instance Fields & Methods][class-fields] | Daniel Ehrenberg                            | Daniel Ehrenberg, Jeff Morrison                 | [:question:](https://github.com/tc39/test262/issues/1161)         |
+|          | [`Function.prototype.toString` revision][function-to-string]                     | Michael Ficarra                             | Michael Ficarra                                 | [:white_check_mark:][tests-1163] |
```

Comment:
not a huge deal, but maybe the names of the footnotes should be `function-to-string-tests` etc; that way they're all consistent, and not pegged to specific PR numbers?

Reply:
Makes sense, will fix!

Reply:
Changed this in 4cecdb1 – definitely better, what do you think?